src/components/Kv/KvChipClassic.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<button class="tw-rounded tw-bg-tertiary tw-p-1 tw-uppercase tw-text-small tw-shrink-0">
+		<slot></slot> X
+	</button>
+</template>
+
+<script>
+export default {
+	name: 'KvChipClassic',
+	props: {
```

Comment:
If you're using slots to pass the visible text I think you can skip passing down any props.